daml-lf/spec/transaction.rst

```diff
@@ -288,8 +290,8 @@ As of version 1, these fields are included:
 
 * ``string`` contract_id
 * `message ContractInstance`_ contract_instance
-* repeated ``string`` stakeholders
-* repeated ``string`` signatories
+* repeated ``string`` stakeholders.
+* repeated ``string`` signatories.
```

Comment:
We don't generally have these full stops; do you want to introduce them as general style?

Reply:
it was by mistake, removed.